### PR TITLE
Add UI test_capsule module to the docs

### PR DESCRIPTION
**Pull request by elyezer**
Also remove an extra space from test_capsule module in order to generate
consistent docs for its functions.

### CODE REVIEW

File: docs/api/tests.foreman.ui.rst [hunk 1]
```diff
@@ -19,6 +19,13 @@
     :members:
     :undoc-members:
 
+:mod:`tests.foreman.ui.test_capsule`
+--------------------------------------------
```
Comment (Ichimonji10) [inline docs/api/tests.foreman.ui.rst]: nitpick: can this line be trimmed down? No worries if you have other things on your plate and want to get this merged, as this is a small thing.

**Comment by Ichimonji10** [comment]
ACK pending comments.

**Comment by elyezer** [comment]
Comments addressed.

**Comment by Ichimonji10** [comment]
:+1:

**Comment by JacobCallahan** [comment]
ACK

**Pull request merged**